repotracker/repotracker.go

```diff
@@ -929,16 +929,16 @@ func createVersionItems(ctx context.Context, v *model.Version, metadata model.Ve
 
 		activateVariantAt := time.Now()
 		taskStatuses := []model.BatchTimeTaskStatus{}
-		if v.Requester == evergreen.RepotrackerVersionRequester && evergreen.ShouldConsiderBatchtime(v.Requester) {
```

Comment:
Removing this check will include trigger versions, is that ok?

Reply:
I'm not sure if it's a workflow anyone uses today but I feel as though both cron and batchtime should be allowed for trigger versions, but a ticket on investigating / confirming that sounds helpful.